### PR TITLE
Alternate score filtering nav for review list

### PR DESCRIPTION
**Pull request by kumar303**
Fixes https://github.com/mozilla/addons/issues/12803

**Before**

screenshot 2019-01-16 17 24 48

<hr>

**After**
screenshot 2019-01-17 15 58 55
screenshot 2019-01-17 15 59 19
screenshot 2019-01-17 15 59 48
screenshot 2019-01-17 16 00 40

### CODE REVIEW

**Comment by willdurand** [comment]
On small screens, the select is not vertically centered (10px vs 12px):

screen shot 2019-01-17 at 11 42 08

File: src/amo/pages/AddonReviewList/styles.scss [hunk 1]
```diff
@@ -38,3 +34,15 @@
   padding: 48px 0;
   text-align: center;
 }
+
+.AddonReviewList-filterByScore {
+  margin-bottom: 12px;
```
Comment (willdurand) [inline src/amo/pages/AddonReviewList/styles.scss]: I think this should be `10px`, see comment in the main PR thread.

**Comment by kumar303** [comment]
Thanks for the review. I totally forgot that we already had a mock for this. It looks way better in the header. Fixed!

**Comment by kumar303** [comment]
Good call. I added wrapping which makes the layout slightly better when text is too long. This isn't the nicest layout but it's probably a good compromise. Viewing wrapped text will likely be uncommon.

screenshot 2019-01-18 11 32 58
screenshot 2019-01-18 11 33 18

**Pull request merged**